Homework6/Game/EventLoop.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace Game;
+public class EventLoop
```

Comment:
```suggestion
﻿namespace Game;
﻿
public class EventLoop
```
И, конечно, надо комментарии